### PR TITLE
(CL) Swap Tool Layout Fixes

### PR DESCRIPTION
**Pull request by JoseRFelix**
### ClickUp Task

[Truncate asset chain name](https://app.clickup.com/t/862k3xxda)
[Can't scroll swap page when swap tool overflowing](https://app.clickup.com/t/862k4647f)

### CODE REVIEW

**Comment by jonator** [comment]
This is also a good chance to investigate how the swap tool changes width when 18 decimals are in the send input box @JoseRFelix

File: packages/stores/src/queries-external/position-performance/index.ts [hunk 1]
```diff
@@ -115,46 +115,38 @@ export class ObservableQueryPositionPerformanceMetrics extends ObservableQueryEx
...
-        .reduce((sum, roi) => sum.add(roi), new RatePretty(0))
-        .quo(new Dec(roiPerCoinDenomArray.length));
+      return new RatePretty(
+        currentPositionValue
+          .toDec()
+          .add(unclamiedRewardsValue.toDec())
+          .add(this.totalEarnedValue.toDec())
+          .sub(this.totalPrincipalValue.toDec())
+          .quo(this.totalPrincipalValue.toDec())
+      );
```
Comment (JoseRFelix) [inline packages/stores/src/queries-external/position-performance/index.ts:149]: @sunnya97 @jonator Can you review this changes that showed up on this PR after rebasing from stage 🙏

Comment (jonator) [reply packages/stores/src/queries-external/position-performance/index.ts]: Yep we changed how we calculate ROI, using $ value instead of token amounts

**Review by jonator — CHANGES_REQUESTED** [review]
For some reason, the swap tool won't load
Screenshot 2023-07-18 at 10 23 37 AM

**Comment by JoseRFelix** [comment]
> For some reason, the swap tool won't load

Weird 🤔 . It's working for me in main and frontier. Do you have some repro steps?

**Review by jonator — COMMENTED** [review]
This may have conflicts with the web worker router stuff

Could you retarget this branch into `jonator/web-worker-router`?

**Review by jonator — APPROVED** [review]

**Pull request merged**